toolkits/global/packages/global-forms/demo/context.json

```
@@ -0,0 +1,234 @@
+{
+	"form1": {
+		"errorIconURL": "../img/error.svg#i-error",
+		"selectIconURL": "../img/chevron-more.svg#i-chevron-more",
+		"errorSummary": {
+			"id": "summary",
+			"title": "Here’s what’s wrong",
+			"errors": [
+				{
+					"id": "your-name-error",
+					"error": "That is not a name, it’s a number"
+				},
+				{
+					"id": "checkbox-terms-error",
+					"error": "You must agree to the terms before continuing"
```

To make consistent with my previous suggestion

```suggestion
					"error": "Agree to the terms to continue"
```